bin/update_timestamp

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+
+require File.expand_path("../../config/environment", __FILE__)
+require "specialist_publisher"
+
+def update
+  $stderr.puts %Q{
+USAGE: #{File.basename(__FILE__)} (all|[format ...])(content_id) (date)
+
+1. available formats:
+
+  #{SpecialistPublisher.document_types.sort.join("\n  ")}
+
+2. content_id:
+	To find content_id, login to specialist-publisher	in production, go to content item page and copy content_id from url.
+
+3. date:
+	Format: yyyy/mm/dd or to specify publishing time: yyyy/mm/dd hh:mm
+	}
+
+  exit(1)
+end
+
+update unless ARGV.any?
+document_type = ARGV[0]
+content_id = ARGV[1]
+date = DateTime.parse(ARGV[2])
+
+raise ArgumentError, "please select document type among available formats provided" unless SpecialistPublisher.document_types.include?(document_type)
+raise ArgumentError, "content_id invalid, see point 2. to retrieve it" unless content_id.length > 30 && content_id.length < 40
+raise ArgumentError, "date format invalid, see point 3 for correct format" unless date.is_a?(DateTime)
+
+repository = SpecialistPublisherWiring.get(:repository_registry).for_type(document_type.to_sym)
+document = repository.fetch(content_id.to_s)
```

Comment:
If the `content_id` needs `#to_s`ing, maybe it'd be best to do it just once when it's pulled out of `ARGV`? It might be that it doesn't need it anyway?